easybuild/easyconfigs/s/subread/subread-1.6.5-GCC-8.3.0.eb

```diff
@@ -0,0 +1,36 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'subread'
+version = '1.6.5'
+
+# HTTPS is not valid
+homepage = 'http://bioinf.wehi.edu.au/featureCounts/'
+description = """featureCounts is a highly efficient general-purpose read summarization program that counts mapped
+reads for genomic features such as genes, exons, promoter, gene bodies, genomic bins and chromosomal locations."""
```

Comment:
homepage/description need fixing?

http://subread.sourceforge.net (https doesn't work?!)

Reply:
copy-paste error, thanks.

Reply:
Binsanity needs featureCounts, which is part of subread apparently. Thus the homepage.